ext/ldap/tests/ldap_set_rebind_proc_trampoline.phpt

```diff
@@ -0,0 +1,63 @@
+--TEST--
+ldap_set_rebind_proc() with a trampoline
```

Comment:
Right, the test seems kinda pointless, but it would also be a shame to get rid of this effort...